### PR TITLE
EDU-1405 Video full screen mode

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-1405

### CODE REVIEW

File: src/components/iteration-panel.js [hunk 1]
```diff
@@ -2,30 +2,30 @@ import React from 'react';
...
-        shape="circle"
-        icon={<LeftOutlined />}
-        disabled={disabled || (!alwaysAllowPreviousClick && selectedItemIndex === 0)}
-        onClick={onPreviousClick}
-        />
       <Tooltip title={t('common:reset')} disabled={disabled}>
         <Button
           shape="circle"
-          icon={<UndoOutlined />}
+          icon={<ChevronLeftPipeIcon />}
```
Comment (adrianaluchian) [inline src/components/iteration-panel.js:15]: [This ticket ](https://educandu.atlassian.net/browse/EDU-1429) made me think that in this context the "reset" button could be confused with redo/replay, so I changed the icons to something hopefully clearer.

File: src/components/media-player/media-player.less [hunk 2]
```diff
@@ -13,6 +13,13 @@
     padding: 0;
     height: 0;
   }
+
+  &.is-fullscreen {
+    display: grid;
+    grid-auto-flow: row;
+    align-items: center;
+    background-color: white;
```
Comment (ahelmberger) [inline src/components/media-player/media-player.less:21]: What do you think about `black` here? Especially when the video takes only half of the screen the big white margins feel a little overpowering to me (maybe I'm just used to having black margins from YouTube, but they seem to catch less attention).

**Comment by ahelmberger** [comment]
Looks and works really well. I also like the `can` -> `allow` renamings :smile:

File: src/components/media-player/media-player.less (hunk 2 above)
Comment (adrianaluchian) [reply src/components/media-player/media-player.less]: True, I agree, I've made it black 👍

**Comment by adrianaluchian** [comment]
> Looks and works really well. I also like the `can` -> `allow` renamings 😄

That gladdens me 😆

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**